src/pypi_changes/_cli.py

```diff
@@ -43,6 +44,18 @@ def _define_cli_arguments() -> ArgumentParser:
     cache_help = "seconds how long requests should be cached (pass 0 to bypass the cache, -1 to cache forever)"
     parser.add_argument("--cache-duration", "-d", default=3600, type=int, help=cache_help, metavar="SEC")
 
+    parser.add_argument(
+        "--sort",
+        "-s",
+        help="Sorting method",
```

Comment:
All other help starts with lowercase, so should this one

Reply:
👍